<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameBaseOSD</class>
 <widget class="QWidget" name="FrameBaseOSD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="FrameTextInput" name="inputHeading" native="true"/>
        </item>
        <item>
         <widget class="FrameTextInput" name="inputPitch" native="true"/>
        </item>
        <item>
         <widget class="FrameTextInput" name="inputRoll" native="true"/>
        </item>
        <item>
         <widget class="FrameOSDMode" name="mode" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameOSDMode</class>
   <extends>QWidget</extends>
   <header>frame_osd_mode.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrameTextInput</class>
   <extends>QWidget</extends>
   <header>frame_text_input.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
